docker/build_autoware.dockerfile

```diff
@@ -42,10 +42,6 @@ RUN git clone https://github.com/autowarefoundation/autoware.git && \
 # install ros-humble-pacmod3-msgs manually because rosdep tries to install ros-galactic-pacmod3-msgs
 # remove gpg because build error happens in ad_api_visualizers for some reasons...
 
-# workaround: remove agnocast because CARET doesn't support Agnocast yet and Agnocast is not used by default
-RUN rm -rf autoware/src/middleware/external/agnocast
-RUN rm -rf autoware/src/universe/autoware_universe/common/autoware_agnocast_wrapper
-
```

Comment:
I see no reason to delete these lines.
If this is an intentional change, please explain why.